bundles/org.eclipse.swt/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-Name: %pluginName
 Bundle-Vendor: %providerName
 Bundle-SymbolicName: org.eclipse.swt; singleton:=true
-Bundle-Version: 3.125.0.qualifier
+Bundle-Version: 3.125.200.qualifier
```

Comment:
`0 -> 200` is wrong. Should be `0 -> 100`.
But also all fragments need to be updated.
Please do it in dedicated PR.